Gist/Gist.swift

```diff
@@ -21,6 +20,8 @@ public class Gist: GistDelegate {
         self.dataCenter = dataCenter
         Logger.instance.enabled = logging
         messageQueueManager.setup()
+        
+        _ = Gist.shared.getMessageView(Message(messageId: ""))
```

Comment:
What is this line of code doing? A comment explaining why this code is here could be needed.